external/Kconfig

```diff
@@ -20,3 +20,4 @@ source "$EXTERNALDIR/WICED/Kconfig"
 source "$EXTERNALDIR/libopus/Kconfig"
 source "$EXTERNALDIR/stress_tool/Kconfig"
 source "$EXTERNALDIR/swepd/Kconfig"
+source "$EXTERNALDIR/esp_idf_port/Kconfig"
```

Comment:
as compre to other directory in external, like WICED,  is it a proper directory name ?